chwall.yml.example

```diff
@@ -8,6 +8,9 @@ bing:
   - de-DE
   - ja-JP
 general:
+  # Desktop environment: gnome, xfce, mate, sway, feh, plasma
+  # For KDE Plasma, use: desktop: plasma
+  # desktop: plasma
```

Comment:
Line 12 is not necessary. Or if you think people might be confused by the word "plasma", use kde as value everywhere (I have no opinion in the matter as I never ever used KDE myself, so I’m pretty far away from all this wording ^^).

On line 13, if you provide an exemple, it must be the default value. So `desktop: gnome`.